server/services/timelineService.ts

```diff
@@ -19,10 +19,12 @@ export default class TimelineService {
     }
   }
 
-  filterTimelineByActionPlanCreatedOrUpdatedEvent = (timelineResponse: TimelineResponse) => {
+  filterTimelineByActionPlanOrGoalEvent = (timelineResponse: TimelineResponse) => {
```

Comment:
My PR will cause a merge conflict here.
I think what you'll want to do is take the version of this file `timelineService.ts` from `main`, and simply add `GOAL_UPDATED` to the array `SUPPORTED_TIMELINE_EVENTS`

Reply:
Sounds good - I can resolve the conflict once your PR is merged